### PR TITLE
Added wedbio

### PR DESCRIPTION
**Pull request by darrylsepeda**
Wedbio.com is a wedding website that specializes in Elegant Wedding Website Design, Invitations, Online RSVP, Guests List, Event Reminder, Reporting, e-Wallet, and more.

Thank you for your work!

### CODE REVIEW

File: _data/sites.json [hunk 1]
```diff
@@ -8181,6 +8181,16 @@
...
+    {
+        "name": "Wedbio.com",
+        "url": "https://wedbio.com",
+        "difficulty": "hard",
+        "notes": "You would need to contact the team by submit contact us form to get your account closed.",
+        "domains": [
+            "wedbio.com"
+        ]
+    },
+
```
Comment (tupaschoal) [inline _data/sites.json:8193]:
```suggestion

```
Remove trailing spaces

File: _data/sites.json [hunk 2]
```diff
@@ -8181,6 +8181,16 @@
         ]
     },
 
+    {
+        "name": "Wedbio.com",
+        "url": "https://wedbio.com",
+        "difficulty": "hard",
+        "notes": "You would need to contact the team by submit contact us form to get your account closed.",
```
Comment (tupaschoal) [inline _data/sites.json:8188]:
```suggestion
        "notes": "You have to contact them via the \"contact us form\" to get your account closed.",
```
Just a suggestion of a rewording. Do you have the link that goes directly to the contact form? That would be very useful!

File: _data/sites.json [hunk 3]
```diff
@@ -8181,6 +8181,16 @@
         ]
     },
 
+    {
+        "name": "Wedbio.com",
```
Comment (tupaschoal) [inline _data/sites.json:8185]:
```suggestion
        "name": "Wedbio",
```
Seems like they advertise their name as just `Wedbio`, without the `.com.`

File: _data/sites.json (hunk 2 above)
Comment (darrylsepeda) [reply _data/sites.json]: it just use an anchor like this `https://wedbio.com/#contact`, btw how do you add suggestion, I can't edit my pull request anymore via github web.

Comment (darrylsepeda) [reply _data/sites.json]: nevermind, have found it. thanks for your suggestion @tupaschoal, please help to review again :)

**Comment by tupaschoal** [comment]
Thanks for your contribution!

**Pull request merged**